src/libraries/System.Net.Security/src/System/Net/Security/SslStreamPal.Unix.cs

```diff
@@ -46,7 +46,7 @@ public static void VerifyPackageInfo()
             return HandshakeInternal(ref context, inputBuffer, ref outputBuffer, sslAuthenticationOptions, clientCertificateSelectionCallback);
         }
 
-        public static SafeFreeCredentials? AcquireCredentialsHandle(SslAuthenticationOptions sslAuthenticationOptions)
```

Comment:
If it's an analyzer driving us to use _1 and _2, why wasn't it previously triggering? sslAuthenticationOptions wasn't used here, either.

Reply:
it's not analyzer related, probably just personal preference. I am not sure who wrote it this way originally, maybe @wfurt ?